bot/processors/filters.py

```diff
@@ -41,7 +41,7 @@ def has_callback_id(callback_id, event):
     if 'callback_id' in event:
         return event['callback_id'] == callback_id
     else:
-        return event['view']['callback_id']
+        return event.get('view', {}).get('callback_id') == callback_id
```

Comment:
in the new bot framework, returning `True` means "pass the original event on", but if you pass a non-boolean, it's assumed that that's a new event to pass on in place of the original one - so the `else` clause effectively replaced the event with the string `event['view']['callback_id']`